src/App.tsx

```diff
@@ -20,12 +33,14 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={query}
+                onChange={(event) => handleChange(event.target.value)}
```

Comment:
```suggestion
                onChange={handleChange}
```
make it work like this